index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <section>
+        <header>
+           <div class="logo">
+            <span>My Logo</a></span>
+           </div>
+           <div class="hamburger">
+            <div class="bar"></div>
+            
+           </div> 
+           <nav>
+            <ul class="nav-list">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+         </nav>
+       </header>
+    </section>
+    <main>
+        <section class="img-bg">
+        <div class="container">
+            <div class="content-wrapper">
+            <div class="main-content">
+                <h1>i'm Amaka Konwea Glad to see you! </h1>
+                <p class="paragraph">I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div>
+                <p class="lets-connect">LET'S CONNECT</p>
```

Comment:
- [ ] the `LET'S CONNECT` should be a title describing the social links below it so kindly replace the `<p>` tag with `<hx>` tag (h2,h3,etc...) 🙏